stack/components/redisinsight.py

```diff
@@ -46,7 +46,7 @@ def generate_url(self, version):
                 f"RedisInsight-{self.APPTYPE}.{osname}-{arch}.tar.gz",
             )
 
-        return f"https://s3.amazonaws.com/redisinsight.download/public/releases/{version}/redisstack/RedisInsight-{self.APPTYPE}-{osname}.{arch}.tar.gz"
+        return f"https://s3.amazonaws.com/redisinsight.download/public/releases/{version}/web/RedisInsight-{self.APPTYPE}-{osname}.{arch}.tar.gz"
```

Comment:
What happens if we ever package not /web? Not that we ever built the app packages, but doesn't this choice limit things?

Reply:
How about now? Or you want me to completely get rid off `RedisInsight` and `RedisInsightWeb` classes?